src/app/aliases/aliases.component.ts

```diff
@@ -55,6 +56,14 @@ export class AliasesComponent extends Base implements OnInit {
           this.router.navigate(['/organizations', collection.organizationName, 'collections', collection.name]);
         }
       });
+    } else if (this.type === 'workflow-versions' && this.alias) {
```

Comment:
Every if statement tests `this.alias`. I would have one `if (this.alias)` at the top so you don't have to keep repeating the test.